pod/main/static/js/main.js

```diff
@@ -1297,3 +1297,10 @@ function flashing(elem, duration) {
     elem.classList.remove("flashing_field");
   }, duration);
 }
+
+// Favorites
+
+const favoriteButtonManage = document.querySelector("#card-managevideo #fav-button");
```

Comment:
je ne comprend pas. pourquoi retirer le bouton en js ?

Reply:
ok en regardant un peu merci de retirer ce code JS. Voici une alternative : 
fichier video_edit.html L.166 `{% include "videos/link_video.html" with video=form.instance hide_favorite_link=True %}`
fichier link_video.html L.6 `{% if USE_FAVORITES and not hide_favorite_link %}`
Merci de tester et de me dire si c'est ok

Reply:
vous pouvez rajouter hide_favorite_link=True à tous les appels que vous souhaitez

Reply:
un peu mieux `{% if USE_FAVORITES and not hide_favorite_link is True %}`

Reply:
En modifiant ces 2 lignes : 
- `{% include "videos/link_video.html" with video=form.instance hide_favorite_link=True %}`
- `{% if USE_FAVORITES and not hide_favorite_link is True %}`

Le bouton apparait toujours pour les pages de complétion, cut...

Reply:
Non c'est bon, je sais !